docs/getting-started/devenv/nodejs-development-tools.md

```diff
@@ -0,0 +1,50 @@
+# Node.js development tools
+
+When developing for the Block Editor, you will need [Node.js](https://nodejs.org/en) development tools along with a code editor and a local WordPress environment. Node.js (`node`) is an open-source runtime environment that allows you to execute JavaScript outside of the web browser.
+
+Installing `node` will automatically include Node Package Manager (`npm`) which includes the script `npx`, two tools you will frequently use in block and plugin development.
```

Comment:
I would rephrase this to mention both aliases and to refer to both of them as tools 

> Installing `node` will automatically include the Node Package Manager (`npm`) and the Node Package eXecute (`npx`), two tools you will frequently use in block and plugin development.